oio/xcute/jobs/blob_rebuilder.py

```diff
@@ -103,12 +106,33 @@ def sanitize_params(cls, job_params):
             job_params.get('allow_frozen_container'),
             cls.DEFAULT_ALLOW_FROZEN_CT)
 
+        set_incident_date = int_value(
+            job_params.get('set_specific_incident_date'),
+            None)
+        if set_incident_date is None:
+            if boolean_value(job_params.get('set_incident_date'),
+                             cls.DEFAULT_DECLARE_INCIDENT_DATE):
+                set_incident_date = int(time.time())
+        if set_incident_date is not None:
+            sanitized_job_params['set_incident_date'] = set_incident_date
```

Comment:
Can we keep two parameters on top of having two flags? I don't really see any reason in favor of merging them, it may be confusing when you show a job, and it will certainly raises questions on the UI side as to why a parameter disappeared after the creation.

Reply:
Indeed, it will be clearer for the user.
```
$ openio-admin xcute rawx rebuild -c config.params.set_incident_date -c config.params.set_specific_incident_date 127.0.0.1:6010
+------------------------------------------+----------------------------------+
| Field                                    | Value                            |
+------------------------------------------+----------------------------------+
| config.params.set_incident_date          | False                            |
| config.params.set_specific_incident_date | None                             |
+------------------------------------------+----------------------------------+
$ openio-admin xcute rawx rebuild -c config.params.set_incident_date -c config.params.set_specific_incident_date 127.0.0.1:6010 --set-incident-date
+------------------------------------------+----------------------------------+
| Field                                    | Value                            |
+------------------------------------------+----------------------------------+
| config.params.set_incident_date          | True                             |
| config.params.set_specific_incident_date | 1585815292                       |
+------------------------------------------+----------------------------------+
$ openio-admin xcute rawx rebuild -c config.params.set_incident_date -c config.params.set_specific_incident_date 127.0.0.1:6010 --set-specific-incident-date 1685815292
+------------------------------------------+----------------------------------+
| Field                                    | Value                            |
+------------------------------------------+----------------------------------+
| config.params.set_incident_date          | True                             |
| config.params.set_specific_incident_date | 1685815292                       |
+------------------------------------------+----------------------------------+
$ openio-admin xcute rawx rebuild -c config.params.set_incident_date -c config.params.set_specific_incident_date 127.0.0.1:6010 --set-incident-date --set-specific-incident-date 1785815292
+------------------------------------------+----------------------------------+
| Field                                    | Value                            |
+------------------------------------------+----------------------------------+
| config.params.set_incident_date          | True                             |
| config.params.set_specific_incident_date | 1785815292                       |
+------------------------------------------+----------------------------------+
```